### PR TITLE
Task/pma 70 common code refactoring

### PR DESCRIPTION
**Pull request by AelxRmoan**

### CODE REVIEW

File: src/components/board/components/column-list/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  border-radius: 50px;
+  border: 3px solid var(--color-gray-200);
+}
+
+.column-wrapper {
+  display: flex;
+}
+
+.add-column-wrapper {
+  opacity: 0.5;
```
Comment (cptShell) [inline src/components/board/components/column-list/styles.module.scss:25]: Pls delete opacity interactions. I think the box shadow is enough

File: src/components/board/components/column-list/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  padding: 24px;
+  display: flex;
+  align-items: center;
+  gap: 24px;
+  cursor: pointer;
+  transition: opacity 0.2s, box-shadow 0.2s;
+}
+
+.add-column-wrapper:hover {
+  opacity: 0.7;
```
Comment (cptShell) [inline src/components/board/components/column-list/styles.module.scss:39]: Pls delete opacity interactions. I think the box shadow is enough

File: src/components/common/button/button.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+type Props = {
+  title: AppLocalizationKey,
+  onClick?: (e?: MouseEvent) => void,
+  className?: string
+};
+
+export const Button: FC<Props> = ({ title, onClick, className }) => {
+
+  return (
+    <button className={`${styles.button} ${className}`} onClick={onClick}>
```
Comment (liza-veis) [inline src/components/common/button/button.tsx]: use clsx for this

File: src/components/common/button/button.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { FC, MouseEvent } from 'react';
+import { FormattedMessage } from '../common';
+import styles from './styles.module.scss';
+import { AppLocalizationKey } from '~/common/types/types';
+
+type Props = {
+  title: AppLocalizationKey,
+  onClick?: (e?: MouseEvent) => void,
```
Comment (cptShell) [inline src/components/common/button/button.tsx]: "e" can be whithout spec symbols

File: src/components/profile/components/edit-form.tsx [hunk 5]
```diff
@@ -75,12 +76,11 @@ export const EditForm: FC = () => {
...
           </div>
-          <button onClick={handleEditable} className={styles['form-button']}>
-            <FormattedMessage
-              as="span"
-              message="profile.editForm.buttons.editUser"
-            />
-          </button>
+          <Button
+            title={'profile.editForm.buttons.editUser'}
+            onClick={handleEditable as (e?: MouseEvent) => void}
```
Comment (liza-veis) [inline src/components/profile/components/edit-form.tsx]:
```suggestion
            onClick={handleEditable}
```

File: src/components/main/main.tsx [hunk 6]
```diff
@@ -4,7 +4,7 @@ import { AppRoute } from '~/common/enums/enums';
 import { useAppSelector, useAppDispatch } from '~/hooks/hooks';
 import { board as boardActions } from '~/store/actions';
 import { ConfirmationModal } from '../common/confirmation-modal/confirmation-modal';
-import { BoardCreatingForm } from './components/board-creating-form';
+import { BoardCreatingForm } from '../common/board-creating-form/board-creating-form';
 import { FormattedMessage } from '../common/common';
 import styles from './styles.module.scss';
 import bucketImg from '~/assets/images/delete-bucket.svg';
```
Comment (cptShell) [inline src/components/main/main.tsx]: [9, 10] = [10, 9]
styles has lowest order

**Review by cptShell — APPROVED** [review]
GGWP easy peasy lemon squizy

**Review by liza-veis — APPROVED** [review]

**Pull request merged**